imgee/storage.py

```diff
@@ -29,17 +29,23 @@ def get_resized_image(img, size, is_thumbnail=False):
     """
     registry = imgee.registry
```

Comment:
This is a circular import. Can you possibly define the registry before other modules are imported, to avoid the circular import?

One way is to move the registry's `__init__` code into a distinct `init_app` method and call that separately after the app is configured (`__init__` also takes an optional `app` parameter and if present calls `init_app`).